### PR TITLE
Longevity test

### PR DESCRIPTION
**Pull request by neha-Gupta1**
Longevity test which runs every night on same repo with the latest release code.  

#### Does this PR need a docs update or release note?

- [ ] :no_entry: No

#### Type of change

- [ ] :broom: Tech Debt/Cleanup

#### Issue(s)

* [#<issue>](https://github.com/alcionai/corso/issues/3679)

#### Test Plan

- [ ] :muscle: Manual

### CODE REVIEW

File: .github/workflows/longetivity_test.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        id: repo-init
+        run: |
+          set -euo pipefail
+          prefix='longevity'
+          echo -e "\nRepo init test\n" >> ${{ env.CORSO_LOG_FILE }}
+          ./corso repo init s3 \
+            --no-stats \
+            --hide-progress \
+            --prefix $prefix \
+            --bucket ${{ secrets.CI_TESTS_S3_BUCKET }} \
```
Comment (ashmrtn) [inline .github/workflows/longetivity_test.yml:89]: I think we may need to adjust the [bucket cleanup script](https://github.com/alcionai/corso/blob/main/.github/workflows/weekly_cleanup.yml). I think right now it deletes all contents in the bucket. If we want a long-lived repo we'll need to exclude this prefix

**Review by ashmrtn — APPROVED** [review]
looks good so far. This can be merged and other additions can be made in separate PRs

**Comment by ashmrtn** [comment]
we should also send notifications about failures here to slack

File: .github/workflows/longetivity_test.yml (hunk 1 above)
Comment (neha-Gupta1) [reply .github/workflows/longetivity_test.yml:89]: I will take care of this in next PR. And also add notification to slack.

**Pull request merged**